roles/username-distribution/templates/username-distribution.yml.j2

```diff
@@ -0,0 +1,157 @@
+---
+kind: List
+apiVersion: v1
+metadata: {}
+items:
+- apiVersion: apps.openshift.io/v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      app: username-distribution
+      app.kubernetes.io/component: username-distribution
+      app.kubernetes.io/instance: username-distribution
+      app.kubernetes.io/part-of: username-distribution-app
+    name: username-distribution
+  spec:
+    replicas: 1
+    revisionHistoryLimit: 10
+    selector:
+      app: username-distribution
+      deploymentconfig: username-distribution
+    strategy:
+      activeDeadlineSeconds: 21600
+      resources: {}
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        annotations:
+          openshift.io/generated-by: OpenShiftWebConsole
+        labels:
+          app: username-distribution
+          deploymentconfig: username-distribution
+      spec:
+        containers:
+        - env:
+          - name: LAB_TITLE
+            value: DevSecOps Workshop - Secure Software Factory
+          - name: LAB_USER_COUNT
+            value: '{{ workshop_users | length }}'
```

Comment:
There's an off-by-one error here due to `workshop_admin`. I think better to use the value of the `number_of_users` variable.